morsecode.rb

```diff
@@ -1,3 +1,20 @@
-LETTERS = { '.-' => 'A',  '-...' => 'B',  '-.-.' => 'C',  '-..' => 'D',  '.' => 'E',  '..-.' => 'F',  '--.' => 'G',  '....' => 'H',  '..' => 'I',  '.---' => 'J',  '-.-' => 'K',  '.-..' => 'L',
-  '--' => 'M',  '-.' => 'N',  '---' => 'O',  '.--.' => 'P',  '--.-' => 'Q',  '.-.' => 'R',  '...' => 'S',  '-' => 'T',  '..-' => 'U',  '...-' => 'V',  '.--' => 'W',  '-..-' => 'X',  '-.--' => 'Y',  '--..' => 'Z'
-} 
\ No newline at end of file
+LETTERS = { '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E', '..-.' => 'F', '--.' => 'G',
+            '....' => 'H', '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N',
+            '---' => 'O', '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T', '..-' => 'U',
+            '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y', '--..' => 'Z' }.freeze
+
+def convert_word(word)
+  arr = []
+  m_word = word.split
+  m_word.each { |w| arr.push(LETTERS[w]) }
+  arr = arr.join
+end
+
+def convert_string(string)
+  arr = []
+  words = string.split('   ')
+  words.each { |word| arr.push(convert_word(word)) }
+  arr = arr.join(' ')
+end
+
+puts convert_string('.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...')
```

Comment:
- [ ] Please add a method to decode a Morse code character, takes a string parameter, and return the corresponding character in uppercase. as mentioned in the project description.

![image](https://user-images.githubusercontent.com/68796232/185265765-de6fd116-d814-4e77-8919-178ee307347c.png)